agent/engine/errors.go

```diff
@@ -255,3 +255,30 @@ func (err CannotListContainersError) Error() string {
 func (err CannotListContainersError) ErrorName() string {
 	return "CannotListContainersError"
 }
+
+// ENIInformationError indicates any error related to eni information
+type ENIInformationError struct {
+	fromError error
+}
+
+func (err ENIInformationError) Error() string {
+	return err.fromError.Error()
+}
+
+func (err ENIInformationError) ErrorName() string {
+	return "ENIInformationError"
+}
+
+// NetworkNamespaceError indicates any error when dealing with the network
```

Comment:
Can you rename this to be more customer friendly? `ContainerNetworkingError` perhaps?